api/modules/automations.js

```diff
@@ -132,6 +132,7 @@ const automations = module.exports = (() => {
         createPayment,
         getUserOrganizations,
         getOrganizationRepos,
+        getClientFromExternalId,
```

Comment:
NIT: 

1. `getClientFromExternalId` should go above `getUserOrganizations` to improve code readability

2. `getXWithId` is more readable/makes more sense semantically than `getXFromId`